documentation/mkdocs.yml

```
@@ -33,6 +33,8 @@ nav:
         - transportRequestCreate: steps/transportRequestCreate.md
         - transportRequestRelease:  steps/transportRequestRelease.md
         - transportRequestUploadFile: steps/transportRequestUploadFile.md
+    - 'Scenarios':
+        - 'Fiori build and deploy': scenarios/ui5-sap-cp/Readme.md
```

If that's not too long I would make sure its understood we're dealing with an MTA build here and Cloud Platform.
```suggestion
        - 'Fiori (MTA) build and deployment to SAP Cloud Platform': scenarios/ui5-sap-cp/Readme.md
```

---

Or another Suggestion:
`UI5 (MTA) build and ...`